debian/control

```diff
@@ -30,9 +30,9 @@ Architecture: all
 Depends: python3 (<< 3.7),
          python3 (>= 3.4),
          python3-pkg-resources,
-         python3-distutils,
          adduser
-Recommends: python3-cryptography
+Recommends: python3-cryptography,
+            python3-distutils
```

Comment:
It's a hard dependency on `18.04`, Kolibri breaks without it... so it's a bit concerning that it's now moved to `recommends`, because this means the package will install without complaining, for instance with `dpkg -i` and then break later.

We should probably move it back and specify a conditional dependency like `python3-distutils | python3-setuptools`

Reply:
In 16.04 etc, `distutils.util` is found in `libpython3.5-stdlib`... I think the rule could be `python3-distutils | libpython3-stdlib`

Reply:
(but I don't know how this is read, it should be "if python3-distutils exists it's a dependency but if it doesn't exist, then libpython3-stdlib is".

Reply:
yes that sounds great! I was thinking about doing it but didn't know the syntax... moving `python3-distutils` is definitely better! 👍

Reply:
Ping me when you have it ready for testing @lyw07! 
Will this enable the `deb` install on Bionic?

Reply:
thank you @radinamatic ! Yes it will enable the deb install on Bionic. It should be ready soon later today

Reply:
Sorry for not mentioning @lyw07 - I have already built it and tested on 16.04 and 18.04, will upload asap

Reply:
@benjaoming thanks! that's great!

Reply:
Eh, nope, having issues on 16.04 with this notation :(

Reply:
I put `python3-stdlib` instead of the correct `libpython3-stdlib` (a lib with a lib, how great)